index.html

```diff
@@ -1,34 +1,204 @@
+                </div>
+
+                <img src="./images/Rectangle 21 (4).png" alt="image"
+                    class="imgCard6">
+
+                <div class="card6">
+                    <h2>Project name goes here</h2>
+                    <div class="buttons">
+                        <div class="projectBtn">
+                            <button type="button"  class="btn1">HTML/CSS</button>
+                            <button type="button"  class="btn2">Ruby on Rails</button>
+                            <button type="button"  class="btn3">JavaScript</button>
+                        </div>
+                        <button type="submit" class="submitBtn">See this project<img
+                                src="./images/Union.png" alt="direction image"
+                                id="arrow"></button>
+                    </div>
+                </div>
+
+                <img src="./images/Rectangle 21 (5).png" alt="image"
+                    class="imgCard7">
+
+                <div class="card7">
+                    <h2>Project name goes here</h2>
+                    <div class="buttons">
+                        <div class="projectBtn">
+                            <button type="button"  class="btn1">HTML/CSS</button>
+                            <button type="button"  class="btn2">Ruby on Rails</button>
+                            <button type="button"  class="btn3">JavaScript</button>
+                        </div>
+                        <button type="submit" class="submitBtn">See this project<img
+                                src="./images/Union.png" alt="direction image"
+                                id="arrow"></button>
+                    </div>
+                </div>
+                <section class="about">
+                    <h1>About</h1>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+                        sed do eiusmod tempor incididunt ut labore et dolore
+                        magna aliqua. Ut enim ad minim veniam, quis nostrud
+                        exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+                    <button type="button"  class="aboutButton">Get my CV <img
+                            src="./images/ic_download.png" alt="this an image"
+                            id="aboutImg"></button>
+
+                    <div class="cards">
+                        <div class="card1">
+                            <a class="heading1" href="#">Languages</a>
+                            <ul class="link1">
+                                <a href=""><li>JavaScript(ES5/ES6)</li></a>
+                                <a href=""><li>Ruby</li></a>
+                                <a href=""><li>HTML</li></a>
+                                <a href=""><li>CSS</li></a>
+                                <img id="hashTag" src="./images/hash.png"
+                                    alt="hashTagImage">
+                            </ul>
```

Comment:
To make your project semantically accurate and improve accessibility, please remove the empty `a` tags wrapping your `li`. As explained [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul), there are the only permitted direct child to `ul` tags 😉:
![image](https://user-images.githubusercontent.com/42799579/198422208-9234b33e-f487-451e-aa44-3b019c30e8cd.png)

Reply:
![Screenshot 2022-10-28 at 13 39 21](https://user-images.githubusercontent.com/37203742/198590441-9291f4c5-c031-471b-9cb9-c84d8f908039.png)